clean.sh

```diff
@@ -3,13 +3,37 @@
 set -e -u
 
 TERMUX_SCRIPTDIR=$(cd "$(realpath "$(dirname "$0")")"; pwd)
+CLEANUP_MESSAGE="Cleaning up build artifacts."
 
 # Store pid of current process in a file for docker__run_docker_exec_trap
 . "$TERMUX_SCRIPTDIR/scripts/utils/docker/docker.sh"; docker__create_docker_exec_pid_file
 
 # Get variable CGCT_DIR
 . "$TERMUX_SCRIPTDIR/scripts/properties.sh"
 
+while (($# >= 1)); do
```

Comment:
```suggestion
while (( $# )); do
```
Numbers other than 0 are truthy.